src/QuadForm/Quad/ZGenus.jl

```diff
@@ -896,6 +896,25 @@ function Base.:(==)(G1::ZZGenus, G2::ZZGenus)
   return true
 end
 
+Base.hash(G::ZZGenus, u::UInt) = xor(hash(G), u)
+
+
+function Base.hash(G::ZZGenus)
```

Comment:
One is only supposed to define `hash(::MyType, ::Uint)`. The 1-argument version is then created by julia Base.